src/features/ui/components/Button/index.ts

```diff
@@ -6,7 +6,38 @@ import { StyleReset } from '../StyleReset'
 
 type ButtonProps = {
   size?: 'small' | 'medium'
-  accent?: 'normal' | 'primary' | 'destructive' | 'edit'
+  accent?: 'normal' | 'primary' | 'destructive' | 'silent'
+}
+
+const accents = {
+  // Default value.
+  normal: null,
+
+  primary: css`
+    --background-color: ${colors.accent.primary};
+    --background-color-hover: ${colors.accent.primaryHover};
+  `,
+
+  destructive: css`
+    --background-color: ${colors.accent.destructive};
+    --background-color-hover: ${colors.accent.destructiveHover};
+  `,
+
+  silent: css`
+    --background-color: ${colors.background.inactive};
+    --text-color: ${colors.text.inactive};
+    --background-color-hover: ${colors.background.inactiveHover};
+  `,
+}
```

Comment:
Thing of objects are ways to improve readability and combine related assets. In this case, we use to highly simplify readability of the Button's CSS.

Unfortunately, styled-components has no better way to declare these variants in a logical order (after the defaults), but it's still easier to reason.